test.sh

```diff
@@ -25,6 +25,15 @@ if [ -n "$TRACE" ]; then
   set -x
 fi
 
+# Enabling specifying KB version by setting KB_VERSION variable in your
+# environment. This is primarily to assist testing version specific functionality
+# like `dep ensure`.
+#
+# $ KB_VERSION=0.1.7 test.sh
+KB_VERSION=${KB_VERSION:-unknown}
```

Comment:
In the Travis testing, do we want to test this against one kb_version or multiple?